### ISSUE TITLE
Object.extname is deprecated.

### ISSUE THREAD

**Issue by rendezvous14** [gliviu/autocomplete-robot-framework#2]
Argument to `path.extname` must be a string
```
Object.extname (/Applications/Atom.app/Contents/Resources/app.asar/src/electron-shims.js:20:10)
buildImportedResource (/Users/NattapolWilarat/.atom/packages/autocomplete-robot-framework/lib/keywords.js:127:26)
Array.map (native)
addKeywords (/Users/NattapolWilarat/.atom/packages/autocomplete-robot-framework/lib/keywords.js:136:44)
Object.addResource (/Users/NattapolWilarat/.atom/packages/autocomplete-robot-framework/lib/keywords.js:113:3)
reloadAutocompleteDataForEditor (/Users/NattapolWilarat/.atom/packages/autocomplete-robot-framework/lib/autocomplete-plus-provider.js:222:20)
```

**Comment by gliviu** [comment]
```
Resource  xx.robot
Resource
Resource  yy.robot
```
will report the error for missing resource path. This can happen while typing resource declarations.

**Comment by gliviu** [comment]
Published v3.1.0 that should fix it.

**Comment by rusnino** [comment]
Hello, It happened again

Argument to path.extname must be a string

```
Object.extname - C:\Users\rufo0618\AppData\Local\atom\app-1.39.1\resources\app.asar\src\electron-shims.js:20:10
getKeywordSuggestions - C:\Users\rufo0618\.atom\packages\autocomplete-robot-framework\lib\autocomplete.js:116:81
Object.getSuggestions - C:\Users\rufo0618\.atom\packages\autocomplete-robot-framework\lib\autocomplete.js:17:28
<unknown> - C:\Users\rufo0618\.atom\packages\autocomplete-robot-framework\lib\autocomplete-plus-provider.js:438:40
Promise - null:null:null
Object.getSuggestions - C:\Users\rufo0618\.atom\packages\autocomplete-robot-framework\lib\autocomplete-plus-provider.js:434:12
```
